keyard/tests/test_integration_store.py

```diff
@@ -2,15 +2,33 @@
 import pytest
 
 from keyard import store
+from keyard import testutils
 
 
-@pytest.mark.xfail
+@pytest.mark.skipif(not testutils.etcd_is_available(), reason="etcd is missing")
```

Comment:
I already have a class to test etcd with a mock, this class is to make sure etcd is responding as I described on the mock version (did find some problems with my mocks because o this tests) and to make sure it really works with etcd as expected